README.md

```diff
@@ -1,29 +1,29 @@
 [![Build Status](https://github.com/ytdl-org/youtube-dl/workflows/CI/badge.svg)](https://github.com/ytdl-org/youtube-dl/actions?query=workflow%3ACI)
 
 
-youtube-dl - download videos from youtube.com or other video platforms
-
-- [INSTALLATION](#installation)
-- [DESCRIPTION](#description)
-- [OPTIONS](#options)
-- [CONFIGURATION](#configuration)
-- [OUTPUT TEMPLATE](#output-template)
-- [FORMAT SELECTION](#format-selection)
-- [VIDEO SELECTION](#video-selection)
+youtube-dl - download videos from youtube.com or other video platforms.
+
+- [Installation](#installation)
+- [Description](#description)
+- [Options](#options)
+- [Configuration](#configuration)
+- [Output template](#output-template)
+- [Format selection](#format-selection)
+- [Video selection](#video-selection)
 - [FAQ](#faq)
-- [DEVELOPER INSTRUCTIONS](#developer-instructions)
-- [EMBEDDING YOUTUBE-DL](#embedding-youtube-dl)
-- [BUGS](#bugs)
-- [COPYRIGHT](#copyright)
+- [Developer instructions](#developer-instructions)
+- [Embedding youtube-dl](#embedding-youtube-dl)
+- [Bugs](#bugs)
+- [Copyright](#copyright)
 
 # INSTALLATION
 
-To install it right away for all UNIX users (Linux, macOS, etc.), type:
+To install it right away for all Unix users (Linux, macOS, etc.), type:
 
     sudo curl -L https://yt-dl.org/downloads/latest/youtube-dl -o /usr/local/bin/youtube-dl
     sudo chmod a+rx /usr/local/bin/youtube-dl
 
-If you do not have curl, you can alternatively use a recent wget:
+If you do not have curl, you can alternatively use a recent version of wget:
```

Comment:
There ain't much of a point in this change.

Reply:
It brings additional context, else its "recent wget" but what is "wget", but with "version of wget" it means oh its version of something. It clarifies and reduces ambiguity as to what "recent" refers to.

Reply:
If the word "curl" won't confuse someone, then "use a recent wget" certainly won't.